Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -172,22 +200,36 @@ - (id)init {
     return self;
 }
 
+@dynamic contactStore;
```

Comment:
Huh. It looks like [the goal posts keep moving](http://stackoverflow.com/a/9368743) on what/when autosynthesis is engaged. Github does not have an emoticon that is glowering quite enough for me to express myself.